skeleton/gitops-template/app-of-apps/application-ci.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: argoproj.io/v1alpha1
```

Comment:
Nitpick: If this is only meant to be used when Tekton is used, maybe it should be named `tekton-ci.yaml` or similarly?

Reply:
Fixed